Tests/TestingApps/PaywallsTester/PaywallsTester/Networking/HTTPClient.swift

```diff
@@ -73,7 +73,11 @@ public final class HTTPClient {
     }
 
     func removeCookies() {
-        self.session.configuration.httpCookieStorage?.removeCookies(since: .distantPast)
+        // the following call only persists the deletion on device, does not work on simulators
+        // self.session.configuration.httpCookieStorage?.removeCookies(since: .distantPast)
```

Comment:
Feels like this would be a weird thing for a maintainer to see without context on why we even keep the old line around, maybe we can make that super clear in the comment like
```suggestion
        // we need to delete cookies one by one because deleting them all directly from 
        // `httpCookieStorage.removeCookies(since:)` only persists the deletion on device
        // and does not work on simulators (FB13733433)
```

Reply:
Committed!